node-persistent-cache.cpp

```diff
@@ -15,7 +15,9 @@
 #include <string.h>
 #include <sys/stat.h>
 #include <sys/types.h>
-#include <unistd.h>
+#ifndef _WIN32
+#    include <unistd.h>
+#endif
 
```

Comment:
This should not be needed. There are many other unistd includes, so cmake just generates empty unistd.h.